dali/pipeline/pipeline.h

```diff
@@ -199,25 +200,6 @@ class DLL_PUBLIC Pipeline {
   }
 
 
-  /**
-   * @brief Sets the external input with the input name to the input data
-   * @tparam Backend
-   * @param name name of the input
-   * @param tl data
-   * @param order synchronization order of the operation - a CUDA stream or host order
-   * @param sync If SetExternalInputHelper should be blocking - waits until provided data is copied
-   *             to the internal buffer
-   * @param no_copy_mode Select whether to use the parameter defined in the External Source or
-   *                     override the mode of operation forcing the copy or no-copy
-   */
-  template <typename Backend>
-  DLL_PUBLIC inline void SetExternalInput(
```

Comment:
Why is this removed? Looks like it is using `TensorVector`, not `TensorList`.

Reply:
Right. But here it's basically the same overload, doesn't really matter which one we remove.